src/components/table/table.html

```diff
@@ -14,14 +14,18 @@
                           :column="column">
                         <div v-if="column.sortable"
                              class="m-table__header-wrap">
-                            <span class="m-table__header-name">{{ column.title }}</span>
+                            <span class="m-table__header-name"
+                                  v-html="column.title"></span>
                             <m-icon-button v-if="column.sortable"
                                            class="m-table__sortable-icon"
+                                           :class="[getColumnSortDirectionClass(column)]"
```

Comment:
Au lieu de faire une méthode qui est réévaluer a chaque rendu (moins performant) c'est mieux de lier les classe a des getter computer

par example : 
`
:class="{ 'm--is-sort-asc': isSortAsc, 'm--is-sort-desc': isSortDesc }"
`

Reply:
La function` getColumnSortDirectionClass(column)` n'est pas une function computed. Elle passe en paramètre la colonne pour pouvoir afficher la bonne classe.